Formula/s-nail.rb

```diff
@@ -26,8 +26,8 @@ def install
     ENV["LC_ALL"] = "C"
     ENV["SOURCE_DATE_EPOCH"] = "844221007"
```

Comment:
Do you still need this line?

Reply:
`ENV["LC_ALL"] = "C"` makes sure that we don't have a build number etc. in the `User-Agent` but that it is generic: `reproducible_build`.

`ENV["SOURCE_DATE_EPOCH"] = "844221007"` makes sure that the date in the mail is equal to the date in the expected output. Otherwise we could not compare the two.

Reply:
Hmm … at least locally, the test seems to work without the `ENV["LC_ALL"] = "C"`